### ISSUE TITLE
Set the default module in a project in Eclipse

### ISSUE THREAD

**Issue by ldcouto** [overturetool/overture#477]
When you have a few modules in a project, it would be nice to set the default module when running it in console mode.

**Comment by peterwvj** [comment]
How would console interaction look like? Could you demonstrate simple usage of this feature?

**Comment by ldcouto** [comment]
Sure. When you launch an SL model in console mode, one of the modules gets set as the default. This is the one that you can operations on directly. There is no way to control what module gets set. There must be some rule for it (maybe it's the last to be edited?).

After launching the console, this default module can be changed using `default [module name]`. This feature would add a selection box (similar to the one for entry point launch configurations) to select the default module.

**Comment by peterwvj** [comment]
Alright I get it. When you execute `default [module name]` the default module does not persist across sessions (it does not edit the launch configuration) - does it?

**Comment by nickbattle** [comment]
I think the "default default" module is just the first in the list of modules build by the parser. In VDMJ, that is the order of the modules in the files that you pass when you start it. I'm not sure what it is in Overture. I think the UI sorts the files i the project alphabetically(?), but that's not to say that they are parsed in the order that they appear on the UI.

**Comment by ldcouto** [comment]
@peterwvj , you are correct. The default has to be re-set every time you launch. Not very convenient when you do edits.

@nickbattle , you're probably right. In VDMJ you can control the submission order but in Overture, you can't. I think it's a fairly simple change to the launch config tab to do this.

**Comment by nickbattle** [comment]
I agree it would be nice. It applies to VDM++ too (there is a default class). The option only makes sense for the Console mode though, so it ought to be active only when that is selected.
